src/ansys/dpf/post/static_mechanical_simulation.py

```diff
@@ -1137,17 +1158,22 @@ def stress_eqv_von_mises_elemental(
             If the problem is multi-stage, can take a list of lists of sector numbers, ordered
             by stage.
         phase_angle_cyclic:
-             For cyclic problems, phase angle to apply (in degrees).
+            For cyclic problems, phase angle to apply (in degrees).
         external_layer:
-             Select the external layer (last layer of solid elements under the skin)
-             of the mesh for plotting and data extraction. If a list is passed, the external
-             layer is computed over list of elements.
+            Select the external layer (last layer of solid elements under the skin)
+            of the mesh for plotting and data extraction. If a list is passed, the external
+            layer is computed over list of elements.
         skin:
-             Select the skin (creates new 2D elements connecting the external nodes)
-             of the mesh for plotting and data extraction. If a list is passed, the skin
-             is computed over list of elements (not supported for cyclic symmetry). Getting the
-             skin on more than one result (several time freq sets, split data...) is only
-             supported starting with Ansys 2023R2.
+            Select the skin (creates new 2D elements connecting the external nodes)
+            of the mesh for plotting and data extraction. If a list is passed, the skin
+            is computed over list of elements (not supported for cyclic symmetry). Getting the
```

Comment:
isn't this description true only for reduce_mesh==True?

Reply:
@cbellot000 do you mean the fact that having `skin=[1, 2, 3]` will map results onto the skin of elements 1, 2, and 3? 

This is still true if `reduce_mesh=False`, where you will get results on "skin elements corresponding to elements 1, 2, and 3 based on the skin generated from the initial mesh". 

If `reduce_mesh=True`, as described in the docstring for it, you will indeed get results on the "all skin elements of the skin generated from the mesh composed of elements 1, 2, and 3 only".